packages/core/src/main/Account.ts

```diff
@@ -299,14 +299,21 @@ class Account extends EventEmitter {
     };
     return Promise.resolve(Root.fromJSON(proto))
       .then((root: Root) => {
+        this.protocolBuffers.Asset = root.lookup('Asset');
         this.protocolBuffers.External = root.lookup('External');
         this.protocolBuffers.GenericMessage = root.lookup('GenericMessage');
         this.protocolBuffers.Text = root.lookup('Text');
       })
```

Comment:
Looks like we should move this to it's own function we can put in the chain and rely on {} or Object assign instead of writing the same over and over again.